### PR TITLE
RET-3720: Add et-wa-post-deployment-ft-tests to authorised services c…

### PR DESCRIPTION
**Pull request by fergusoshea**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RET-3720

### Change description ###
Add et-wa-post-deployment-ft-tests to authorised services ccd-data-store-api on aat for preview

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by Itay-Kirsh — APPROVED** [review]

File: apps/ccd/ccd-data-store-api/aat.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ spec:
...
+        DATA_STORE_S2S_AUTHORISED_SERVICES: ccd_data,ccd_gw,ccd_ps,probate_backend,divorce_ccd_submission,sscs,sscs_bulkscan,cmc,cmc_claim_store,cmc_claim_external_api,jui_webapp,pui_webapp,bulk_scan_orchestrator,fpl_case_service,iac,finrem_ccd_data_migrator,finrem_case_orchestration,employment_tribunals,ethos_repl_service,ccpay_bubble,ctsc_work_allocation,em_ccd_orchestrator,xui_webapp,bulk_scan_payment_processor,pcq_consolidation_service,em_npa_app,ecm_consumer,aac_manage_case_assignment,am_role_assignment_service,wa_task_configuration_api,divorce_frontend,wa_task_management_api,wa_workflow_api,em_hrs_api,nfdiv_case_api,civil_service,wa_task_monitor,ccd_case_document_am_api,adoption_cos_api,adoption_web,et_cos,et_msg_handler,prl_cos_api,wa_case_event_handler,et_sya_api,ds_ui,prl_citizen_frontend,hmc_cft_hearing_service,prl_cos_api,fis_hmc_api,ccd_next_hearing_date_updater,civil_general_applications,fis_cos_api,sptribs_case_api,et-wa-post-deployment-ft-tests
```
Comment (rajaram-nagarajan) [inline apps/ccd/ccd-data-store-api/aat.yaml:21]: As per standard, please use underscore instead of hyphen in the service name.

**Review by rajaram-nagarajan — CHANGES_REQUESTED** [review]

File: apps/ccd/ccd-data-store-api/aat.yaml (hunk 1 above)
Comment (fergusoshea) [reply apps/ccd/ccd-data-store-api/aat.yaml:21]: Thanks Rajaram. I have updated this.

Comment (rajaram-nagarajan) [reply apps/ccd/ccd-data-store-api/aat.yaml:21]: Still I can see the service name (et-wa-post-deployment-ft-test) with hyphen

Comment (fergusoshea) [reply apps/ccd/ccd-data-store-api/aat.yaml]: Sorry, got that change pushed now

**Review by rajaram-nagarajan — APPROVED** [review]
LGTM

**Pull request merged**